### DISCUSSION TITLE
Podcast index

### DISCUSSION THREAD

**Discussion by jcorporation** [jcorporation/myMPD#1340] [Ideas]
This seems to be reasonable to integrate it in myMPD: https://podcastindex.org/

**Comment by Emilot** [comment]
> This seems to be reasonable to integrate it in myMPD: https://podcastindex.org/

Does it have any valuable content ? From a quick search i didn't find something...

PS. I was really sorry that Bandcamp do not provide API. It would be amazing to hear my or any music from it :(
